### PR TITLE
edit README

### PR DESCRIPTION
**Pull request by WhatTheDickens**
Please have a look and see what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# pull-request-practice
\ No newline at end of file
+# Pull request practice
+
+A repository for practicing pull requests. Request away!
```
Comment (cdl-geneseo) [inline README.md]: Is this a detailed enough description?

**Review by ghost — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (WhatTheDickens) [reply README.md]: I thought it would be enough. You want more detail? Like what?

Comment (cdl-geneseo) [reply README.md]: Like who's doing the practicing here and why.

Comment (WhatTheDickens) [reply README.md]: Okay, I'll add more detail. Thanks for the suggestion.

Comment (WhatTheDickens) [reply README.md]: Okay I've made some changes

**Comment by cdl-geneseo** [comment]
Okay this is much better. May need additional changes but I'm going to pull it in for now.

**Pull request merged**